bool reset_odom
---
bool result